### PR TITLE
DP-1.4: QoS Interface Output Queue Counters - Removing SchedulerInputParamsUnsupported deviation and adding scheduler weight limit deviation

### PR DESCRIPTION
**Pull request by charantejag504**
DP-1.4: QoS Interface Output Queue Counters
1. Removing SchedulerInputParamsUnsupported deviation as the issue is fixed
2. Using a new deviation introduced in https://github.com/openconfig/featureprofiles/pull/1350 related to limit the scheduler input weights to 100, since values above 100 are not supported by Junos. In current Junos implementation, the largest weight supported for a queue is 100.

### CODE REVIEW

**Comment by sachendras** [comment]
Please update the description, otherwise LGTM

**Review by sachendras — DISMISSED** [review]

File: feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go [hunk 1]
```diff
@@ -310,6 +310,11 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		}
 	}
 
+	nc1InputWeight := uint64(200)
+	if deviations.SchedulerInputWeightLimit(ondatra.DUT(t, "dut")) {
```
Comment (greg-dennis) [inline feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:314]: No need to lookup the DUT when you already have a reference to it. Just "if deviations.SchedulerInputWeightLimit(dut) {" works.

File: feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go [hunk 2]
```diff
@@ -310,6 +310,11 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
...
+		nc1InputWeight = uint64(100)
```
Comment (greg-dennis) [inline feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:315]: I don't see where SchedulerInputWeightLimit is documented to mean "use an input weight of 100"

**Review by greg-dennis — CHANGES_REQUESTED** [review]

File: feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go (hunk 2 above)
Comment (charantejag504) [reply feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:315]: Please refer https://github.com/openconfig/featureprofiles/pull/1350 where the description of deviation is updated.

File: feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go (hunk 1 above)
Comment (charantejag504) [reply feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go]: addressed

**Review by greg-dennis — APPROVED** [review]
Approving for Go quality

**Review by sachendras — DISMISSED** [review]

File: feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go (hunk 2 above)
Comment (sachendras) [reply feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go:315]: Is this okay for NC1 and AF4 to have weight 100?

**Review by sachendras — APPROVED** [review]

**Pull request merged**